cmd/test/main.go

```diff
@@ -352,7 +378,7 @@ func test(ctx *TestCommandContext, paths []string, options *TestCommandOptions)
 	return nil
 }
 
-func evaluate(ctx *TestCommandContext, filesPaths []string, options *TestCommandOptions) (*ValidationManager, *cliClient.CreateEvaluationResponse, evaluation.ResultType, error) {
+func evaluate(ctx *TestCommandContext, filesPaths []string, options *TestCommandOptions, policy policy_factory.Policy) (*ValidationManager, int, evaluation.FormattedResults, string, error) {
```

Comment:
the functions' return values are unclear. we will need to be more specific about what it returns. adding `int` or `string` alongside other variables is making the code less readable. maybe add them to a struct?